src/pineko/cli/check.py

```diff
@@ -71,13 +71,14 @@ def sub_scvar(grid_path, scale, max_as_order, max_al_order):
     )
     sv_as = None
     central_as = None
-    # TODO: fix this
-    if max_as_effective == max_as_order:
+    # Remember that we have a different definition for the alpha, i.e. we need to sum 1
+    if max_as_effective + 1 == max_as_order:
```

Comment:
Exactly, this is the `+ 1` I was thinking about yesterday.

But maybe I'd move on the other side, since it will be the same `- 1` that you have in the other place.